src/app/children/health-checkup/health-checkup.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, OnInit } from '@angular/core';
+import { HealthCheck } from './health-check';
+import { ColumnDescription } from '../../ui-helper/entity-subrecord/column-description';
+import { ActivatedRoute } from '@angular/router';
+import { EntityMapperService } from 'app/entity/entity-mapper.service';
+import { ChildrenService } from '../children.service';
+import {DatePipe} from '@angular/common';
+
+
+@Component({
+  selector: 'app-health-checkup',
+  templateUrl: './health-checkup.component.html',
+  styleUrls: ['./health-checkup.component.scss']
+})
+
+export class HealthCheckupComponent implements OnInit {
+  records = new Array<HealthCheck>();
+  /**
+   * Column Description for the SubentityRecordComponent
+   * The Date-Column needs to be transformed to apply the MathFormCheck in the SubentityRecordComponent
+   */
+  columns: Array<ColumnDescription> = [
+    new ColumnDescription('date', 'Date', 'date', null,
+    (v: Date) => this.datePipe.transform(v, 'yyyy-MM-dd')),
+    new ColumnDescription('height', 'Height', 'number'),
```

Comment:
at least the labels here need to state the unit (i.e. "in cm" and "in kg")

Reply:
done